pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -74,7 +74,7 @@ func (admitter *PodEvictionAdmitter) Admit(ar *admissionv1.AdmissionReview) *adm
 		!vmi.IsMarkedForEviction() &&
 		vmi.Status.NodeName == launcher.Spec.NodeName &&
 		!dryRun {
-		if err := admitter.markVMI(ar, vmi.Name, vmi.Status.NodeName); err != nil {
+		if err := admitter.markVMI(vmi.Namespace, vmi.Name, vmi.Status.NodeName); err != nil {
```

Comment:
this refactoring should be part of a unique refactoring commit of the `markVMI` function.

Reply:
Done.